R/estimate_KM.R

```diff
@@ -194,9 +148,18 @@ estimate_KM <- function(
       rlang::set_names(strata)
   }
 
+  # Return ------------------------------------------------------------------
```

Comment:
Error in attr(data[[x]], "label") %||% x : could not find function "%||%" Can you look into this? Perhaps we need to increase rlang version?